movieApp/movieApp/Scenes/Main/MainViewController.swift

```diff
@@ -8,46 +8,75 @@
 import UIKit
 
 class MainViewController: UIViewController {
+    struct Constants {
+        static let largeTitleBarText: String = "Movies"
+        static let titleBarText: String = "Popular Movies"
+        static let cellIdentifier: String = "customFilmCell"
+        static let limitForBarShowing: CGFloat = 107
+        static let tableViewRowHeight: Double = 120
+    }
     // MARK: - IBOutlet
     @IBOutlet private weak var filmTableView: UITableView!
+    @IBOutlet private weak var tableViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var headlineLabel: UILabel!
 
     // MARK: - Properties
-    var viewModel: MainViewModel!
+    private var viewModel: MainViewModel!
+    private var count: Double = 10
 
     // MARK: - Life Cycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        title = "Moviessss"
-//        navigationController?.navigationBar.
-//        navigationController?.title = "Movies"
-//        navigationController?.navigationBar.barTintColor = UIColor.vibrantBlue
-        filmTableView.delegate = self
-        filmTableView.dataSource = self
+        setupUI()
+        setupUITableView()
+        setupUIScrollView()
         viewModel = MainViewModel()
-        // Do any additional setup after loading the view.
     }
-    /*
-     // MARK: - Navigation
 
-     // In a storyboard-based application, you will often want to do a little preparation before navigation
-     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-     // Get the new view controller using segue.destination.
-     // Pass the selected object to the new view controller.
-     }
-     */
+    // MARK: - Business Logic
+    func setupUI() {
+        view.backgroundColor = UIColor.whiteTwo
+        headlineLabel.font = UIFont.headline2
```

Comment:
No need to adjust font, text color in code. You can adjust on storyboard

Reply:
fixed